packages/composite-checkout/src/lib/payment-methods/eps.js

```diff
@@ -0,0 +1,263 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from '@emotion/styled';
+import debugFactory from 'debug';
+import { sprintf } from '@wordpress/i18n';
+import { useI18n } from '@automattic/react-i18n';
+
+/**
+ * Internal dependencies
+ */
+import Field from '../../components/field';
+import Button from '../../components/button';
+import {
+	usePaymentProcessor,
+	useTransactionStatus,
+	useLineItems,
+	renderDisplayValueMarkdown,
+	useEvents,
+} from '../../public-api';
+import { SummaryLine, SummaryDetails } from '../styled-components/summary-details';
+import { useFormStatus } from '../form-status';
+import { registerStore, useSelect, useDispatch } from '../../lib/registry';
+import { PaymentMethodLogos } from '../styled-components/payment-method-logos';
+
+const debug = debugFactory( 'composite-checkout:eps-payment-method' );
+
+export function createEpsPaymentMethodStore() {
+	debug( 'creating a new eps payment method store' );
+	const actions = {
+		changeCustomerName( payload ) {
+			return { type: 'CUSTOMER_NAME_SET', payload };
+		},
+		changeCustomerBank( payload ) {
```

Comment:
This store has two data fields: the name and the bank, but down below I only see an input for the name. I think we can remove all mentions of "bank" from this file.